django/views/templates/default_urlconf.html

```diff
@@ -1,44 +1,84 @@
+        </style>
```

Comment:
I'd like to avoid committing compiled CSS. Given the intended use case -- locally accessing a very small page -- there's no need to optimize. Can we keep the CSS in source form?